CMakeLists.txt

```diff
@@ -14,6 +10,10 @@ add_definitions(
 	/D_CRT_SECURE_NO_WARNINGS # disable warnings about old libc functions
 )
 set(CMAKE_CXX_FLAGS "/GR- ${CMAKE_CXX_FLAGS}") # turn off C++ RTTI
+set(CMAKE_C_FLAGS_RELEASE "/GL /Gw")
+set(CMAKE_CXX_FLAGS_RELEASE "/GL /Gw")
+set(CMAKE_EXE_LINKER_FLAGS_RELEASE "/OPT:REF /OPT:ICF")
```

Comment:
It looks like `/OPT:REF` and `/OPT:ICF` is already the default linker flag.

https://learn.microsoft.com/en-us/cpp/build/reference/opt-optimizations?view=msvc-170

Reply:
OK, removed.